Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/GifViewController.swift

```diff
@@ -0,0 +1,383 @@
+//
+//  EmojiMainViewController.swift
+//  StreamChatUI
+//
+//  Created by Mohammed Hanif on 26/04/22.
+//
+
+import UIKit
```

Comment:
@mdhanif-simformsolutions  Please remove `import UIKit`. GiphyUISDK already imported the same

Reply:
Updated this. thank you.